services/install-piwigo.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+function install {
+  # create service directory
+  mkdir -p /srv/piwigo/config
+
+  # create yml(s)
+  cat << EOF > /srv/piwigo/piwigo.yml
+version: "3.7"
+services:
+  piwigo:
+    image: linuxserver/piwigo
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Europe/London
+    volumes:
+      - /srv/piwigo/config:/config
+    ports:
+      - 9093:80
+    restart: unless-stopped
+    depends_on:
+      - db
+  db:
+    image: jsurf/rpi-mariadb
+    restart: unless-stopped
+    ports:
+      - 3306:3306
+    environment:
+      - MYSQL_ROOT_PASSWORD=\${MYSQL_ROOT_PASSWORD_VAR}
+      - MYSQL_DATABASE=\${MYSQL_DATABASE_VAR} 
+    volumes:
+      - /srv/piwigo/datadir:/var/lib/mysql
+EOF
+
+  # add autorun
+  cat << EOF > /srv/piwigo/autorun
+piwigo_autorun=true
+
+if [ "$piwigo_autorun" = true ]; then
+  treehouses services piwigo up
+fi
+EOF
+}
+
+# environment var
+function uses_env {
+  echo "MYSQL_ROOT_PASSWORD_VAR=my-secret-pw"
+  echo "MYSQL_DATABASE_VAR=piwigo" 
+} > /srv/piwigo/.env
+
```

Comment:
The actual .env file creation should be inside `install` and `uses_env` should only `echo true` or `echo false`